server/src/modules/clubs/mappers/club-map.ts

```diff
@@ -1,21 +1,78 @@
 import { UniqueEntityID } from '../../../shared/domain/unique-entity-id'
 import { ClubEntity } from '../../../shared/infra/db/entities/club.entity'
-import { Club } from '../domain/entities/club'
-import { ClubDTO } from './club-dto'
+import { EventEntity } from '../../../shared/infra/db/entities/event.entity'
+import { BasicEvent, Club } from '../domain/entities/club'
+
+interface BasicEventDTO {
+  name: string
+  startTime: string
+  endTime: string
+  backgroundImageURL?: string
+  tags: Array<string>
+}
+
+export interface ClubDTO {
```

Comment:
I think it was fine to keep this interface in the `club-dto.ts` file, it feels out-of-scope in a file about data transformations

Reply:
Okay, I reverted to having DTO files for both clubs and events.